### DISCUSSION TITLE
Create in one-line command: model, migration and seeder

### DISCUSSION THREAD

**Discussion by tuliovargas** [adonisjs/core#2752] [Help] [answered]
Hi everyone!!
How I can create in one-line command: model, migration and seeder?
I've tried it: 
`node ace make:model City -mcs`
but o got the error: **[ error ]  E_INVALID_FLAG: Unknown flag "s"**

**Comment by RomainLanz** [comment, marked as answer]
Hey @tuliovargas! 👋 

You cannot.

There is already an issue requesting this feature.
➡️  https://github.com/adonisjs/lucid/issues/675
